posthog/hogql_queries/web_analytics/stats_table.py

```diff
@@ -438,6 +343,124 @@ def to_path_bounce_query(self) -> ast.SelectQuery:
         assert isinstance(query, ast.SelectQuery)
         return query
 
+    def _main_inner_query(self, breakdown, *, include_session_properties=False):
+        query = parse_select(
+            """
+SELECT
+    any(person_id) AS filtered_person_id,
+    count() AS filtered_pageview_count,
+    {breakdown_value} AS breakdown_value,
+    session.session_id AS session_id,
```

Comment:
A few notes on this:
* include_session_properties was previously passed in so that we could do a performance optimisation, where if there were no session properties, we could avoid joining on the sessions table altogether. This meant we had to group by `events.$session_id` rather than `session.session_id`.
* This optimisation meant that queries with an event property breakdown (e.g. the geoip and device ones) would be a little bit faster
* With how you're comparing to previous periods using the session start timestamp, we're always using the session table, so it's no longer a useful distinction, that performance optimisation can never happen any more
* Hypothetically, we could bring back the performance optimisation, by using the event timestamp in `period_aggregate` rather than the session start timestamp, for queries that don't touch the sessions table
* IMO we don't need to bring back that performance optimization in this PR. First, let's see what the performance is like for loading all queries on the page in one go

Reply:
Done, I've removed the optimization from most queries. The ones that run when no conversion goal is set have been kept.